recipes/granola.qmd

```diff
@@ -0,0 +1,34 @@
+---
+title: "Granola"
+image: "Add the URL to the image here"
+image-alt: "Add the URL to the image here"
```

Comment:
Please add the same weblink for the image.

Reply:
Done - but same question as above.